lib/Capnp/Message.hs

```diff
@@ -492,17 +496,18 @@ allocInSeg msg segIndex size = do
 alloc :: WriteCtx m s => MutMsg s -> WordCount -> m (WordPtr (MutMsg s))
 alloc msg size@(WordCount sizeInt) = do
     when (sizeInt > maxSegmentSize) $
-        throwM E.SizeError
+        throwM $ E.SizeError $ "alloc: sizeInt > maxSegmentSize (" ++ show sizeInt ++ " > " ++ show maxSegmentSize ++ ")"
```

Comment:
I'm hazarding a guess this was initially put in during debugging?

I'm in favor of adding more info to `SizeError`, but exposing the `sizeInt` variable name seems likely confusing. Really it would be better to supply the information as a structured data type rather than an unstructured string.